extensions/index.md

```diff
@@ -4,17 +4,83 @@ title: Extensions
 show_sidebar: true
 ---
 
+## Status
+
+**Extensions are an experimental feature** and should be considered a work
+in progress. There is no official support for extensions in the base JSON
+API specification.
+
+## Extending JSON API <a href="#extending-json-api" id="extending-json-api" class="headerlink"></a>
+
 JSON API can be extended in several ways:
 
-* The `supported-ext` and `ext` media type parameters can be used to negotiate
-  support for extensions,
-  [as discussed in the base specification](/format#extending).
-  Official and custom extensions to the specification are discussed below.
+* [Official](#official-extensions) and [custom](#custom-extensions) extensions
+  are in development. The `supported-ext` and `ext` media type parameters can
```

Comment:
Can we start the sentence that says "The `supported-ext` and `ext`..." with the clause "Currently, ", to suggest that the parameters may change?